### DISCUSSION TITLE
Publishing a Javascript Project In NPM & GithubProjects @ The same time

### DISCUSSION THREAD

**Discussion by varunsridharan** [community/community#25493] [Actions]
<p>Hi All,</p>
<p>Well i am new to github packages. and i am not sure whats the best process.<br>
as of now i publish all my javascript packages in <a href="http://npmjs.com" rel="noopener nofollow ugc">npmjs.com</a> .</p>
<p>i would love to use Github Packages. but my question here is</p>
<p>Should i publish a single package in both package managers ?</p>
<ol>
<li>Github Packages</li>
<li>NPM</li>
</ol>
<p>is that a good process ?</p>

**Comment by jcansdale** [comment, marked as answer]
<blockquote>
<p>Should i publish a single package in both package managers ?</p>
</blockquote>
<p>You should publish public packages to both GitHub and npm.</p>
<p>Try something like this:</p>
<pre><code class="lang-auto">npm publish --registry=https://npm.pkg.github.com
npm publish --registry=https://registry.npmjs.org
</code></pre>
